### PR TITLE
Release candidate

### PR DESCRIPTION
**Pull request by jverkoey**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,27 @@
+# 60.3.0
+
```
Comment (yarneo) [inline CHANGELOG.md:2]: blurb for release.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,27 @@
...
+## New features
+
+MDCNavigationBar has a new flag that, once enabled, allows you to set a font with any size.
```
Comment (yarneo) [inline CHANGELOG.md]: what's the flag, how do i use it?

**Review by yarneo — DISMISSED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (jverkoey) [reply CHANGELOG.md:2]: Done.

File: CHANGELOG.md (hunk 2 above)
Comment (jverkoey) [reply CHANGELOG.md]: Done.

**Review by yarneo — APPROVED** [review]

**Pull request merged**